### DISCUSSION TITLE
Balance fader for fading seamlessly between mute and solo for your own level in the mix

### DISCUSSION THREAD

**Discussion by dingodoppelt** [jamulussoftware/jamulus#1068] [Ideas]
I'm playing multiple instruments on jamulus and found that I need to hear myself at different levels for different instruments. (ranging from louder than everybody else to A LOT LOUDER than everybody else, depending on the instrument I play)
I currently have to set a new client level, then hit ctrl+l to reset all faders to that new client level and redo my mix again.
Since I never touched the "Pan" fader because it doesn't cover my usecase (I send a stereo signal from only one instrument at a time) I thought about converting it into a "balance" fader which would let me adjust my level independently of my set mix for the other clients. (middle position = default, top position = same as "solo" on my fader, bottom position = same as "mute" on my fader) Moving the balance fader above the middle position would attenuate everybody elses levels, moving it below middle would just attenuate my level.
Is it clear what I'm driving at?

**Comment by gene96817** [comment]
I would never need your scenario, although I could see using a similar scenario. Would this generalization work for you.

Consider being able to save N mix profiles.
If you are using mix profile j and then select mix profile k, the client would fade from j to k.

I would then same a profile for different jamming groups (or ensembles). Or, for a rehearsal group, I could have a different profile or quartet A  or quartet B. Currently, I don't know of a way to save mix settings.

**Reply by dingodoppelt** [reply]
I think you can already save the mixer setting AFAIK. Have a look at the file menu.
Loading pre-saved mixes is of no use for me because naturally you don't always play with the same people on public servers.
What I could do is set the new client level to 50 % - including mine - and then go from there depending on the instrument I play... need to think about that
